### PR TITLE
spingboot api quickstart

### PR DESCRIPTION
**Pull request by lyleschemmerling**
API quickstart for springboot

### CODE REVIEW

File: astro/src/content/quickstarts/quickstart-springboot-api.mdx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+description: Quickstart integration of a Spring Boot api application action as a resource server with FusionAuth
+navcategory: getting-started
+prerequisites: Maven, Java
+section: api
+technology: Spring Boot
+language: Java
+icon: /img/icons/spring.svg
+faIcon: fa-leaf
+color: green
+---
```
Comment (bhalsey) [inline astro/src/content/quickstarts/quickstart-springboot-api.mdx:12]: You can add `cta: EmailListCTA` here to get the email CTA

**Comment by bhalsey** [comment]
We've made an attempt at organizing the Rails on Golang quickstarts around the journey categories.  Rails didn't map perfectly, and I think we'll circle back on this.
They also both have Next Steps at the end (the Rails one has links to FA docs).

**Comment by mooreds** [comment]
LGTM :)

File: astro/src/content/quickstarts/quickstart-springboot-api.mdx [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+<Diagram />
+
+While the access token is acquired via the Login API above, this is for simplicity of illustration. The token can be, and typically is, acquired through one of the [OAuth grants](/docs/v1/tech/oauth/).
+
+## Getting Started
+
+In this section, you’ll get FusionAuth up and running and create a resource server which will serve the API.
+
+### Clone the Code
```
Comment (bradmccarty) [inline astro/src/content/quickstarts/quickstart-springboot-api.mdx:39]: Clone The Code, capital T, for consistency.

File: astro/src/content/quickstarts/quickstart-springboot-api.mdx [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `Authorization` header in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+Make a directory for this API. 
+
+```
+mkdir spring-api && cd spring-api
+```
+
+### Get the Template
```
Comment (bradmccarty) [inline astro/src/content/quickstarts/quickstart-springboot-api.mdx:89]: Same here

**Pull request merged**